### ISSUE TITLE
C# support

### ISSUE THREAD

**Issue by Trafo** [Snapchat/djinni#135]
Thx for continue developing djinni, we would like to switch from the dropbox version to your version. But sadly, you dropped the C# support. Any reasons for that? And is it an option, that we add it again? What were the main reasons for removing it?

**Comment by paulocoutinhox** [comment]
This fork never had the C# support. The C# support you find in this fork: 
https://github.com/cross-language-cpp/djinni-generator

**Comment by Trafo** [comment]
Yes, I know, but the original lib had it. And we are not only looking for C# support. We like the direction this lib is going with the future and promise for example and on the top wasp integration is very interesting for us.
And I like to use one project instead of multiple. Was there a reason of dropping C# support?
And is this lib open for C# support, provided by others?

**Comment by paulocoutinhox** [comment]
Again, the original project don't have C# support:
https://github.com/dropbox/djinni/tree/master/src/source

**Comment by LiFengSC** [comment]
Hi @Trafo ,

Thanks for your interest on this project. The original Dropbox version did not have C# support.  And it is very unlikely that a Snap engineer will spend time on developing C# support here.  This is because the main goal of this repo is to support cross-platform development of Snap's mobile and web apps.  So far Snap has no need for C# or other .Net framework languages.  That said, if you, or someone else are willing to contribute and keep maintaining C# support, in a way that does not risk compatibility or stability of the other supported languages, then we will consider merging it and making it available to people with similar needs.
